terraform-tests/aurora_mysql_test.go

```diff
@@ -28,7 +28,7 @@ var _ = Describe("Aurora mysql", Label("aurora-mysql-terraform"), Ordered, func(
 		"cluster_instances":                      3,
 		"serverless_min_capacity":                nil,
 		"serverless_max_capacity":                nil,
-		"engine_version":                         nil,
+		"engine_version":                         "8.0.mysql_aurora.3.02.0",
```

Comment:
**question**: Why have we decided to bring this in now?

Reply:
Good question. Definitely, this is my mistake I just added. I will change it because that is not the default value. Thank you @jhvhs